rdr_client/hpo_assigner.py

```diff
@@ -64,10 +65,10 @@ def main(client):
       if hpo == 'UNSET':
```

Comment:
Seems like we may wan to delete providerLink, awardee, hpoId, site, and organization before the if here. That way we're only passing up what we want to change?

Also, the _get_old_hpo logic has to change to use pairing... we're now not necessarily changing awardees, and we need to show the change in organization or site if appropriate.

Reply:
not done